src/js/pagination/paging-collection.js

```diff
@@ -48,6 +48,8 @@
                 pageSize: 'page_size',
                 sortKey: 'order_by',
                 order: 'sort_order',
+                totalPages: null,
```

Comment:
Yeah, for some reason backbone paginator believes that the client should be telling the server `totalRecords` and `totalPages` - https://backbone-paginator.github.io/backbone.paginator/api/#queryParams

I agree that there's no reason for the client to request `totalRecords` or `totalPages`.